ci-operator/step-registry/osd-ccs/e2e/aws/osd-ccs-e2e-aws-workflow.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "osd-ccs/e2e/aws/osd-ccs-e2e-aws-workflow.yaml",
```

Comment:
BTW, the `e2e` folder hierarchy is not necessary. Installation workflow does not need to indicate it's for e2e or upgrade.

Reply:
Yes, but what is a better `folder` name to store the workflow?